vhdbuilder/packer/cleanup.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash -x
 
+# deadline for old packer resource group deletion is 24 hours
+(( packerRgExpirationInSecs = 24 * 60 * 60 ))
```

Comment:
this is fun new syntax

I always thought this only worked like var=$(( arithmetic * foo ))

Reply:
yes very fancy syntax